modules/mobile/spec/request/observations_spec.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'support/mr_client_helpers'
+require 'medical_records/client'
+require_relative '../support/helpers/sis_session_helper'
+
+RSpec.describe 'observations', type: :request do
+  include MedicalRecords::ClientHelpers
+  include SchemaMatchers
+
+  let!(:user) { sis_user(icn: '32000225') }
+
+  before do
+    allow(MedicalRecords::Client).to receive(:new).and_return(authenticated_client)
+  end
+
+  context 'Premium User' do
```

Comment:
Also, I don't think this context is needed either here or in the test above. The mhv_account_type is not being used in the current setup, but also I don't think it provides any value without other mhv account type contexts. It's not clear to me if mhv account type has any impact because the user you've created is not an mhv user and the tests still pass, so I have to suspect this is just not relevant.

Reply:
No worries, happy to clean it up